spec/features/user_signs_out_spec.rb

```diff
@@ -22,7 +22,7 @@
 
     expect(page).to have_content('Signed in successfully')
 
-    click_link 'Sign Out'
+    click_link 'Sign Out', match: :first
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.